### ISSUE TITLE
ASCII table needs update

### ISSUE THREAD

**Issue by FeralChild64** [MEGA65/mega65-user-guide#43]
The ASCII table in Appendix C is valid for C64, but not for C65 ROMs. For example, CTRL+G produces char 7 - if you execute ?CHR$(7), you'll hear a bell. ?CHR$(8) to disable character switch does not work on C65 ROMs - probably it's used for tabulator, like on the C128. They might be other differences, I wouldn't be surprised if F9-F14 had their own codes.

**Comment by sausagejohnson** [comment]
Thanks @FeralChild64 yep it was put together with C64, 128 and 65 resources, if you find more, please let me know.

**Comment by FeralChild64** [comment]
Well, CHR$(2) for underline works too, you can use CTRL+O for flashing characters - I suppose the C128 PETSCII (https://www.commodore.ca/manuals/128_system_guide/app-i.htm) is a much better base for this table.

**Comment by gardners** [comment]
Also, we currently call the appendix ASCII codes, when it is actually PETSCII codes.  I'll rename the appendix for now.

**Comment by gurcei** [comment]
Ok, I looked over the C128 reference you gave, looks like @Edilbert has handled most of this already, they look really similar now.

I've added the missing CTRL-O shortcut you mentioned into list of CTRL-shortcuts.

Think the only aspect that you mentioned to be chased up is whether their our CHR$() codes for F9-F14, I'm unsure of this as yet.

**Comment by gurcei** [comment]
Ok, done my homework and discovered F9-F14 + HELP values.

Will just proof-read and compare everything against C128, just to assess if there's anything else we could potentially add.

**Comment by gurcei** [comment]
Just to preserve my attempt, I noticed that the C128 page mentioned that char 131 could be injected into the keyboard buffer in order to perform a `RUN`.

I tried doing this for the c65, using the uart monitor, with these commands:
```
s2b0 83   (add 131 to the keyboard-buffer)
sd0 1   (set keyboard buffer length to 1)
```
Didn't seem to perform any run action for the C65 (whereas if I used `s2b0 07` and `sd0 1`, I would hear the bell).

So I'm going to conclude this trick isn't available for the c65/m65.
